utils/goutils_test.go

```diff
@@ -41,71 +41,47 @@ rsc.io/sampler:v1.3.0
 
 func TestGetProjectRoot(t *testing.T) {
 	wd, err := os.Getwd()
-	if err != nil {
-		t.Error(err)
-	}
-	defer os.Chdir(wd)
+	assert.NoError(t, err)
+	defer func() {
+		assert.NoError(t, os.Chdir(wd))
+	}()
```

Comment:
```suggestion
	defer assert.NoError(t, os.Chdir(wd))
```

Reply:
Why? we change back to the wd as a form of cleanup.